src/iframe-attributes.js

```diff
@@ -57,6 +60,14 @@ export function getContextMetadata(
       href: locationHref,
     },
     startTime,
+    tagName: element.tagName,
+    mode: getModeObject(),
+    canary: isCanary(parentWindow),
+    hidden: !viewer.isVisible(),
+    initialIntersection: element.getIntersectionChangeEntry ?
```

Comment:
isn't "A4A ad element" the same as `amp-ad` element? It should have this method. are you saying this method does not exist in runtime? or just in test?